### DISCUSSION TITLE
How to fix this?

### DISCUSSION THREAD

**Discussion by httpanand** [community/community#9490] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/87514488/147843350-bb40ab75-3d7e-41c8-b99c-6ebc4c40a0c9.png)

**Comment by g1f1** [comment]
More informations please?
In what OS distro and browser happen?
You can use a browser extension and write html/css to fix this.

**Reply by httpanand** [reply]
Its fixed by github itself !

**Reply by g1f1** [reply]
ok this is good
